src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -747,8 +744,12 @@ export class CalciteSlider implements LabelableComponent {
         prop = closerToMax || position > this.maxValue ? "maxValue" : "minValue";
       }
     }
-    this[prop] = this.clamp(position, prop);
+    this.lastDragPropValue = this[prop];
     this.dragStart(prop);
+    const thumbHovered = !!this.el.shadowRoot.querySelector(".thumb:hover");
```

Comment:
Can you store the hovered state on the instance using the mouseenter/mouseexit event on the thumb? Querying through the shadowRoot each time isn't very efficient.

Reply:
I was going to write a comment to request this as well.  It also allows easier discoverability as to where the event listener is wired up to respond to mouse events on the thumb.

Reply:
Changed the query to look for `:active `state rather than hover keeping mobile in mind. 
cc: @driskull , @eriklharper